.devcontainer/setup.sh

```diff
@@ -9,3 +8,8 @@ echo "build --config=clang" >> user.bazelrc
 #echo "build --symlink_prefix=/" >> ~/.bazelrc
 
 [[ -n "${BUILD_DIR}" ]] && sudo chown -R "$(id -u):$(id -g)" "${BUILD_DIR}"
+
+echo "Fetching LLVM toolchain..."
+bazel fetch @llvm_toolchain//...
+sudo ln -s "$(bazel info output_base)"/external/llvm_toolchain_llvm/bin/{clang-format,clangd} /usr/local/bin/
+sudo ln -s "$(bazel info output_base)"/external/llvm_toolchain/bin/cc_wrapper.sh /usr/local/bin/
```

Comment:
I would suggest
```
sudo ln -s "$(bazel cquery --output=files @llvm_toolchain//:clangd)" /usr/local/bin/
sudo ln -s "$(bazel cquery --output=files @llvm_toolchain//:clang-format)" /usr/local/bin/
sudo ln -s "$(bazel cquery --output=files @llvm_toolchain//:bin/cc_wrapper.sh)" /usr/local/bin/
```
to minimize hardcoded bits of path (not sure if there's a better target for cc_wrapper.h).

Also possibly instead of the fetch, `bazel build @llvm_toolchain//:clangd` etc. would be more resilient to change, in case the target files at some point become [partially] generated rather than directly fetched.

Reply:
TBH, in this specific case, hard code the binary path is not that bad and I cannot think of any drawbacks except one day we changed our bazel startup options (which may never happens)

Reply:
> That looks better, but did it work for you? For me, it doesn’t print the full paths.

It produces relative paths from the bazel root, which seems to be the same as the working directory vscode uses, which is also the working directory when the script runs (e.g. `clang-path=".devcontainer/clang-wrapper.sh"` works.)
If a full path is needed I think you could use `$(pwd)/$(bazel cquery ...)`